CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Crashes on Windows are now also reported with a popup in addition to stderr
 - Windows: New configuration field `enable_experimental_conpty_backend` which enables support
     for the Pseudoconsole API (ConPTY) added in Windows 10 October 2018 (1809) update
+- New key action `SpawnNewInstance` for launching another instance of Alacritty
```

Comment:
I've initially called this just `action`, but attached the `key` to make it more obvious what it's talking about. But it's absolutely correct that this also applies to mouse bindings. I feel like it might be confusing to call this `binding action` since not everyone knows what that would mean, but `New key and mouse action ...` should do the job.